fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1325,10 +1325,11 @@ analyzeColumnClause
     | qualifiedName  (',' qualifiedName)*                       #regularColumns
     | ALL COLUMNS                                               #allColumns
     | PREDICATE COLUMNS                                         #predicateColumns
+    | MULTI_COLUMNS '(' qualifiedName  (',' qualifiedName)* ')' #combinedMultiColumns
```

Comment:
personally prefer `compositeMultiColumns` or `multiColumnSet`

Reply:
nice, I will modify it at next patch